enable/tools/hover_tool.py

```diff
@@ -149,5 +149,4 @@ def _is_in(self, x, y):
             return any((t, b, r, l))
 
     def _create_timer(self, event):
-        klass = toolkit_object("timer.timer:Timer")
-        self._timer = klass(self.hover_delay, self.on_timer)
+        self._timer = Timer(self.hover_delay, self.on_timer)
```

Comment:
Actually, on second thoughts,
```suggestion
        self._timer = do_after(self.on_timer, self.hover_delay)
```
is probably slightly preferable (see https://github.com/enthought/pyface/blob/master/pyface/timer/do_later.py#L45), and should work across new and old versions of Pyface timers.

If you do this, you will need to remove the call to `self._timer.Stop()` in `on_timer`

Reply:
Thanks for all of this guidance, @corranwebster. Unfortunately, the direct replacement with `do_after` didn't work quite as hoped, since the [old version of `do_after`](https://github.com/enthought/pyface/blob/14c26736ebc3e245cd8e6d469d995d8265873f81/pyface/timer/do_later.py#L33-L45) didn't actually return the timer instance, so `self._timer` stayed `None`.

I think it's probably safe to remove the `_timer` attribute entirely and create a new timer on each hover event, but I wasn't sure, so I opted on the side of changing as little as possible and set `self._timer` to be a `DoLaterTimer` instance.

I have a few more test cases I'd like to add to flesh out the coverage on the conditionals.